site/en/guides/build/index.md

```diff
@@ -84,45 +42,44 @@ locations:
 Type | Location
 ---- | ----
 Compile-time constants | Listed in all the header files in [`/src/core/config`](https://github.com/openthread/openthread/tree/main/src/core/config)
-Makefile build switches | Listed in [`/examples/common-switches.mk`](https://github.com/openthread/openthread/tree/main/examples/common-switches.mk)
+Makefile build switches | Listed in [`openthread/examples/README.md`](https://github.com/openthread/openthread/blob/main/examples/README.md)
 
 > Note: Each example platform included in OpenThread specifies some, but not all, of the constants and flags that the platform supports. Modify the example platform's `/openthread-core-{platform}-config.h` file to enable or disable compile-time constants prior to building.
 
 ### Build examples
 
 Use a switch to enable functionality for an example platform. For example, to
-build the CC2538 example with Commissioner and Joiner support enabled:
+build the binary for the CC2538 platform with Commissioner and Joiner support enabled:
 
 ```
-$ make -f examples/Makefile-cc2538 COMMISSIONER=1 JOINER=1
+$ ./script/build -DOT_COMMISSIONER=1 -DOT_JOINER=1
 ```
 
 Or, to build the nRF52840 example with the [Jam Detection
 feature](/guides/build/features/jam-detection) enabled:
 
 ```
-$ make -f examples/Makefile-nrf52840 JAM_DETECTION=1
+$ ./script/build nrf52840 UART_trans  -DOT_JAM_DETECTION=1
 ```
 
 ### Binaries
 
-The following binaries are generated in `/output/{platform}/bin` from the build process. To determine which binaries are generated, use configure option flags with the `./configure` command to generate an updated `Makefile` for building. For example, to build OpenThread and generate only the CLI binaries:
+The following binaries are generated in `/build/bin` from the build process. To determine which binaries are generated, use flags with the `script/build` command. For example, to build OpenThread and generate only the FTD CLI binary:
```

Comment:
should we use the relative path instead, e.g.`./build/bin/`, `./script/build`?

Reply:
I'm not very sure about the convention here. I thought that the former version was using absolute paths.

Reply:
I think relative paths make more sense. @Vyrastas ?

Reply:
> I'm not very sure about the convention here. I thought that the former version was using absolute paths.

Here are the build output and script directory, both are relative to the source repo directory, e.g,  
you may find the full path for build output would be `<path-to-src repo>/build`